src/pal/src/misc/sysinfo.cpp

```diff
@@ -31,9 +31,12 @@ Revision History:
 
 #if HAVE_SYSCONF
 // <unistd.h> already included above
-#elif HAVE_SYSCTL
+#endif
+#if HAVE_SYSCTL && not defined(__linux__)
```

Comment:
Can you please use `!` instead of `not` here and below?

Reply:
updated.